src/task/MonitorTask.ts

```diff
@@ -306,18 +307,41 @@ export default class MonitorTask
       }
     );
 
-    this.applyHealthPolicy(
-      this.sendingAudioFailureHealthPolicy,
-      connectionHealthData,
-      () => {
-        const attributes = this.generateBaseAudioVideoEventAttributes();
-        this.context.eventController?.publishEvent('sendingAudioFailed', attributes);
-      },
-      () => {
-        const attributes = this.generateBaseAudioVideoEventAttributes();
-        this.context.eventController?.publishEvent('sendingAudioRecovered', attributes);
-      }
-    );
+    if (this.shouldEvaluateSendingAudioFailurePolicy) {
+      this.applyHealthPolicy(
+        this.sendingAudioFailureHealthPolicy,
+        connectionHealthData,
+        () => {
+          const attributes = this.generateBaseAudioVideoEventAttributes();
+          this.context.eventController?.publishEvent('sendingAudioFailed', attributes);
+        },
+        () => {
+          const attributes = this.generateBaseAudioVideoEventAttributes();
+          this.context.eventController?.publishEvent('sendingAudioRecovered', attributes);
+        }
+      );
+    }
+  }
+
+  audioVideoDidStart(): void {
+    // We want to wait until we're connected to avoid false-positives.
+    this.shouldEvaluateSendingAudioFailurePolicy = true;
```

Comment:
nit: Could generalize this flag to something like `isMeetingConnected` as it can be used in other places.